CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ If you’re only fixing a bug, it’s fine to submit a pull request right away b
 
 ## Contribution Prerequisites
 
-- You have the latest version of [Node](https://nodejs.org), [Git](https://git-scm.com) and [Firebase CLI](https://firebase.google.com/docs/cli) installed
+- You have the latest version of [Node](https://nodejs.org), [Git](https://git-scm.com), [Java](https://www.java.com/en/) _(to be used for Firebase emulators)_ and [Firebase CLI](https://firebase.google.com/docs/cli) installed
```

Comment:
@ux-git: thanks for updating this one. But would you please adjust the Java to `[OpenJDK](https://www.oracle.com/java/technologies/downloads/)`.